docs/developer/data-plane-signaling/data-plane-signaling.md

```diff
@@ -39,25 +39,112 @@ The data plane access token may be renewable based on the capabilities of its as
 
 ### The Signaling Protocol
 
-All requests must support idempotent behavior. Data planes must therefore perform request de-duplication. After a data plane commits a request, it will return an ack to the control plane, which will transition the `TransferProcess` to its next state (e.g., `STARTED`, `SUSPENDED`, `TERMINATED`). If a successful ack is not received, the control plane will resend the request during a subsequent tick period.
+All requests must support idempotent behavior. Data planes must therefore perform request de-duplication. After a data plane commits a request, it will return an ack to the control plane, which will transition the `TransferProcess` to its next state. If a successful ack is not received, the control plane will resend the request during a subsequent tick period.
 
-##### 1. `START` 
```

Comment:
The original document structure was focussing on the flow, not the messages. A proposal would be to keep this document as it was and create a new one that focusses on EDC's implementation:

Possible title could be `data-plane-mechanisms.md`

Possible ToC:
   1. State Machine
   3. State `STARTED`
   4. State `SUSPENDED`
   5. State `TERMINATED`
   6. State `COMPLETED`
   7. etc.

For each state, it is described how the state is reached (e.g., via a message or data plane internal processes (i.a., in case of errors)), how the message looks like, what Http endpoint is provided and called, and some further comments (those can be adopted from this PR's content). There is no need to apply the structure of the DSP documents. This removes the "specification character". As stated in the discussion and the PR, we do **NOT** want to create a specification for that in any EDC repositories.

Reply:
> how the message looks like

We would prefer inline JSON snippets of EDC messages, no `.ttl` files or `.puml` diagrams.

Reply:
WDYT? @jimmarino

Reply:
Agreed documentation is fine but not a specification. There are also issues that need to be corrected related to sync vs. async messaging and the state machine.

Why can't this be shifted to: How to implement a Data Plane? This can be described in a language-neutral way and focus on the signaling API that needs to be implemented.  The emphasis would be on describing the HTTP endpoints that need to be implemented.